_pkgdown.yml

```
@@ -5,7 +5,12 @@ destination: reference
 url: https://pkgs.rstudio.com/quillt/
 
 template:
-  package: quillt
+  package: tidytemplate
+  bootstrap: 5
+  bslib:
+    primary: "#096B72"
+    navbar-background: "#e6f3fc"
```

```suggestion
    navbar-background: "#e6f3fc"
  trailing_slash_redirect: true
```

Needed for redirects through `pkgs.rstudio.com` to work when the trailing slash is missing

---

Thanks!!

---

Oh how did I missed that. Thanks @gadenbuie !